### PR TITLE
Refactor readme

### PR DESCRIPTION
**Pull request by godofdream**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,3 +1,33 @@
+Copyright (c) 2017 Go-siris team. All rights reserved.
```
Comment (Dexus) [inline LICENSE]: Can you please change this to Go-SIRIS team?

File: README.md [hunk 2]
```diff
@@ -171,179 +159,77 @@ func createUser(ctx context.Context) {
...
-
-Documentation
------------
-
- <a href="https://github.com/go-siris/siris/tree/master/_examples#table-of-contents"><img width="155" src="https://raw.githubusercontent.com/iris-contrib/website/gh-pages/assets/book/cover_4.jpg"></a>
+# Feature Overview
+
+| Feature | More Information |
+| --------|------------------|
+| high performance | TODO add to [speedtest](https://github.com/smallnest/go-web-framework-benchmark) |
```
Comment (Dexus) [inline README.md]: Is already done. Only a test run need to be done.

File: README.md [hunk 3]
```diff
@@ -171,179 +159,77 @@ func createUser(ctx context.Context) {
...
+| Limits | RequestBody TODO add docu|
+| Localization | [i18n](_examples/intermediate/i18n) |
+| Logger Engines | [file-logger](_examples/beginner/file-logger) [request-logger](_examples/beginner/request-logger) |
+| Routing | [routing](_examples/beginner/routing)  |
+| Sessions | [sessions](_examples/intermediate/sessions) |
+| Static Files | [file-server](_examples/beginner/file-server) |
+| Subdomains and Grouping | [subdomains](_examples/intermediate/subdomains) |
+| Tempalte HTML, django, pug(jade), handlebars, amber | [views](_examples/intermediate/view) |
+| Tooling | Typescript integration + Web IDE |
+| Updates | Checks for Updates |
```
Comment (Dexus) [inline README.md]: Looks like this is not implemented... or was never...

**Review by Dexus — DISMISSED** [review]
At last we should use a "code of conduct" and a "contibution" markdown. To perform the GITHUB state of the art recommendations for contributions etc.

And at last good work!

**Comment by Dexus** [comment]
A example for the [contributing.md](https://github.com/blog/1184-contributing-guidelines) i like: https://github.com/haraka/Haraka/wiki/Contributing

**Comment by Dexus** [comment]
Via https://github.com/go-siris/siris/community/code-of-conduct/new?template=contributor-covenant you can create a new code-of-conduct.md

**Comment by Dexus** [comment]
Can you also create a contributing.md?

File: contributing.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+git checkout -b new_branch
+$EDITOR file_to_change
+git add file_to_change
+git commit # with a useful message
+git push origin new_branch
+```
+
+The `git commit` step(s) will launch you into `$EDITOR` where the first line should be a summary of the change(s) in less than 50 characters. Additional paragraphs can be added starting on line 3.
+
+To submit new_branch as a Pull Request, visit the [Haraka project page](https://github.com/go-siris/siris) where your recently pushed branches will appear with a green "Pull Request" button.
```
Comment (Dexus) [inline contributing.md]: SIRIS project page

File: contributing.md [hunk 5]
```diff
@@ -0,0 +1,62 @@
+We love and appreciate contributions to Go-Siris.
```
Comment (Dexus) [inline contributing.md]: Go-SIRIS ;)

**Review by Dexus — CHANGES_REQUESTED** [review]

**Comment by Dexus** [comment]
When done with the changes, it can be pushed to branch devel.
Better will be with two PR, where the vendor changes are splitted from the "refactor readme".

Regards, Dexus (Josef)

File: contributing.md (hunk 4 above)
Comment (godofdream) [reply contributing.md]: aah damn XD

**Comment by godofdream** [comment]
see https://github.com/go-siris/siris/pull/18

File: contributing.md [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+On branches with more than a couple commits, it's usually best to squash the commits (condense them into one) before submitting the change(s) as a PR. Notable exceptions to the single commit guideline are:
+
+* where there are multiple logical changes, put each in a commit (easier to review and revert)
+* whitespace changes belong in their own commit
+* no-op code refactoring is separate from functional changes
+
+To rebase:
+
+```sh
+git remote add haraka https://github.com/go-siris/siris.git
```
Comment (Dexus) [inline contributing.md:42]: DAMN, `haraka` need to be `siris`?

Comment (godofdream) [reply contributing.md:42]: sorry i have overseen this one, i will fix it on devel

Comment (godofdream) [reply contributing.md:42]: fixed in https://github.com/go-siris/siris/commit/38b22d38ba9247a346f02c1cd53d046483f70500

**Pull request closed without merging**